### DISCUSSION TITLE
[Feature Request] Option to notify users for the digital release date of movies if the movie was added to the collections

### DISCUSSION THREAD

**Discussion by 32423d** [trakt/showly#622] [Ideas / Feature Requests]
Several movies are now initially releasing in theaters and then on digital streaming platforms on a later date. It would be nice if there is an option to make the app send reminder notifications if a movie had a digital release and if it was added to the collections. I am not sure if this feature would be possible but Trakt.tv does list release dates for both theatrical releases and digital releases for movies on their website but I am not sure if the data can be called from the Trakt.tv API.

**Comment by michaldrabik** [comment]
@32423d It can be called from the API but only for a single Movie. In that case I doubt I will be implementing this along with notifications as this would mean that the whole collection of movies would need to be monitored and API-called separately to check if new date appears/changes.

**Comment by Bamboozul** [comment]
@michaldrabik @32423d In this case, I believe having the option to display a theatrical release date or a digital release date will be very useful.

**Comment by tomeczeka** [comment]
Perhaps this could be a thing to enable on a movie-by-movie basis for daily API checks? Perhaps include a warning to not overuse it for the sake of app performance and serverside costs? Meh, still sounds risky -- if Trakt finds Showly users overuse API they may block it

**Comment by michaldrabik** [comment]
The information about theatrical and digital release will probably make it to the app sooner or later. Just not along with scanning whole collection and notifying.

**Comment by dr0vur** [comment]
I would love this as a filter in "Watchlist" and "Discover" sections, to show only those results that have a digital release. I personally i'm not interested in the notifying part.
